src/cargo/ops/cargo_new.rs

```diff
@@ -366,6 +461,10 @@ fn existing_vcs_repo(path: &Path, cwd: &Path) -> bool {
     GitRepo::discover(path, cwd).is_ok() || HgRepo::discover(path, cwd).is_ok()
 }
 
+fn file(p: &Path, contents: &[u8]) -> io::Result<()> {
+        try!(File::create(p)).write_all(contents)
```

Comment:
I also think there may be a helper for this in `cargo::util::paths`? If not, could you add one there?